src/main/java/com/wavefront/proxy/ProxyForwarderImpl.java

```diff
@@ -52,15 +52,16 @@
   private final Counter numCounterEventReceived;
   private final Counter numContainerMetricReceived;
   private final Counter metricsSendFailure;
-  private final Wavefront wavefront;
+  private final WavefrontProxyClient wavefront;
```

Comment:
can you call this wavefrontSender or wavefrontProxyClient instead of wavefront ?